{{cookiecutter.module_name|slugify}}/Makefile

```
@@ -0,0 +1,51 @@
+pyupgrade = pyupgrade --py39-plus $$(find inmanta_plugins tests -type f -name '*.py')
```

```suggestion
pyupgrade = pyupgrade --py311-plus $$(find inmanta_plugins tests -type f -name '*.py')
```
Min version we support is iso7